conda/core/repodata.py

```diff
@@ -591,7 +585,7 @@ def maybe_decompress(filename, resp_content):
         raise CondaIndexError("Invalid index file: {0}: {1}".format(join_url(url, filename), e))
 
 
-def make_feature_record(feature_name, feature_value):
+def make_feature_record_key_value(feature_name, feature_value):
```

Comment:
It's not clear why we should even keep this here; I know we could possibly bring it back, but the easiest way to do that would be to effectively revert this entire PR